### ISSUE TITLE
Color markers by age

### ISSUE THREAD

**Issue by brentfraser** [nickfox/GpsTracker#51]
For the last reported location in a route, use different markers for "recent"  and "old".  Set the cutoff as a constant in the Javascript.  The database procedures/views would have to be modified to calculate the age (e.g. age = 20 minutes) of the location.

**Comment by nickfox** [comment]
Yeah, I could see this as a useful enhancement. It would be best to do this on a different branch and then test it on websmithing.com (where we get huge amounts of traffic) and will be able to see results quickly.
